Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// Unity doesn't include the required assemblies (i.e. the ones below).
+// Given that the .NET backend is deprecated by Unity at this point it's we have
+// to work around this on our end.
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Threading.Tasks;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class NearInteractionTouchableTests
+    {
+        /// <summary>
+        /// Test creating adding a NearInteractionTouchable to GameObject programmatically.
+        /// Should be able to run scene without getting any exceptions.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator Test01_ManipulationHandlerInstantiate()
+        {
+            var testObject = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            testObject.transform.localScale = Vector3.one * 0.2f;
+
+            var touchable = testObject.AddComponent<NearInteractionTouchable>();
+            // Wait for two frames to make sure we don't get null pointer exception.
+            yield return null;
+            yield return null;
+
+            GameObject.Destroy(testObject);
+            // Wait for a frame to give Unity a change to actually destroy the object
+            yield return null;
+        }
+
+        private InputSimulationService inputSim;
+        IMixedRealityInputSystem inputSystem;
+
+        [SetUp]
+        public void SetupMrtk()
+        {
+            TestUtilities.InitializeMixedRealityToolkitAndCreateScenes(true);
+            TestUtilities.InitializePlayspace();
+
+            inputSim = PlayModeTestUtilities.GetInputSimulationService();
+            Assert.NotNull(inputSim);
+
+            Assert.IsTrue(MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem));
+            Assert.NotNull(inputSystem);
+        }
+
+        [TearDown]
+        public void ShutdownMrtk()
```

Comment:
Should we do this in all our tests?

Reply:
I've copied this from the ManipulationHandlerTests, not sure if it's strictly necessary. Scene loading and running multiple tests sequentially runs fine now, so i guess it destroys the MRTK instance properly without the test doing that explicitly. Will check with @googlan to confirm.

Reply:
We do, because our default setup routine initializes playspace, and we need to reset it before the next test. If you don't reset it, it _won't always fail_ the next test, but it might fail some asserts, which rely on camera position.